### ISSUE TITLE
404 on BottomSheetBehavior

### ISSUE THREAD

**Issue by feinstein** [romannurik/AndroidSDKSearchExtension#67]
The `BottomSheetBehavior` link returns a 404:

https://android.googlesource.com/platform/frameworks/support/+/refs/heads/master/design/src/android/support/design/widget/BottomSheetBehavior.java

I believe this source code is not under the main Android Repository, but here:

https://github.com/material-components/material-components-android/blob/master/lib/java/com/google/android/material/bottomsheet/BottomSheetBehavior.java

**Comment by JakeWharton** [comment]
Yeah the material stuff shouldn't even be displayed on d.android.com anymore. They should have their own docs like any other Google-proprietary stuff like Play Services and Firebase. I'll file a bug for that.

**Comment by JakeWharton** [comment]
Looks like there's a bunch of proprietary stuff on d.android.com like the Play Billing library so the line is blurred...

**Comment by feinstein** [comment]
Thanks Jake!
